### DISCUSSION TITLE
Mobile Bug after username change

### DISCUSSION THREAD

**Discussion by nu11pointer** [community/community#69544] [Mobile]
After changing the username in the Github website, the mobile app, that was previously logged in with the old username, still stays logged in, but several tasks such as adding a starred repo into a list stops working, stating that the username does not exist.
The application should log out when the username is changed or these bugs should be fixed, since the app still thinks my username is the old one.

**Comment by Mohammed71o** [comment]
> After changing the username in the Github website, the mobile app, that was previously logged in with the old username, still stays logged in, but several tasks such as adding a starred repo into a list stops working, stating that the username does not exist.
> The application should log out when the username is changed or these bugs should be fixed, since the app still thinks my username is the old one.

Tank you Boys.😁

**Reply by Mohammed71o** [reply]
class   MainActivity : componentActivity(){
    override  fun  onCreate ( savedInstanceState : Bundle ) { 
         super . onCreate ( savedInstanceState )setConent {
                  GreetingCardTheme {
                            // A surface container using the 'background' color from the theme
                             Surface (
                                   modifier = modifier . fillMaxSide ()
                                     color = MaterialTheme color background
() {
                                      Greeting ( "Android") un
                                } 
                           } 
                    } 
              } 
       } 


CC4.

**Comment by kauaneiras** [comment]
🚀
